vsts_extension/Tasks/HydraLabDeployTest/hydra-lab-deploy-test.ts

```diff
@@ -10,20 +10,43 @@ async function run() {
     try {
         tl.setResourcePath(path.join( __dirname, 'task.json'));
 
+        var workingDirectory: string | undefined = tl.getInput('workingDirectory');
+        if (!workingDirectory){
+            workingDirectory = tl.getVariable('SYSTEM_DEFAULTWORKINGDIRECTORY');
+        }
+        if (!workingDirectory){
+            workingDirectory = __dirname;
+        }
+
+        fs.mkdir(workingDirectory, { recursive: true }, (err) => {
+            if (err) throw err;
+            console.log('##[section]Working Directory Folder created successfully!'); 
+        });
+
+        tl.cd(workingDirectory);
+        console.log('##[section]Working Directory: %s', workingDirectory);
+
+        var build = false;
```

Comment:
Could we rename this variable as isBuild or isBuildPipeline?

Reply:
Renamed